CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+3.3.0
+=====
+
+*   (internal) Bump minimum PHP version to ^7.4.
+*   (internal) Upgrade to `becklyn/rad`.
```

Comment:
I would mark both of them as improvement.

Upgrading is (generally) making things better.